demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -936,7 +936,7 @@ def generate_single_command_section(
             ]
         )
         for arg in arguments:
-            description = arg.get("description")
+            description = arg.get("description", "")
```

Comment:
Not sure it should be an empty string. Maybe better to have some value like "Missing argument description" or "No description provided"?